src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services9.yml

```diff
@@ -76,6 +76,10 @@ services:
         class: stdClass
         decorates: decorated
         decoration_inner_name: decorated.pif-pouf
+    deprecated_service:
+        class: stdClass
+        deprecated: true
```

Comment:
Then in PHP we would still have one (or two) arguments : `$container->setDeprecated(true, 'message template %service_id');`

But I think that in PHP, it should be okay to allow more customization, unlike in XML or YAML. I'll make the change then, it would be easier to allow more customization rather than restrict it afterwards.